patches/components-permissions-android-permission_dialog_delegate.cc.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/components/permissions/android/permission_dialog_delegate.cc b/components/permissions/android/permission_dialog_delegate.cc
+index 19905a0a9f24afdfe4bd6f78d03be2a633d18cba..c23e1979383c9bb0f68f89774598e9aec0a44228 100644
+--- a/components/permissions/android/permission_dialog_delegate.cc
++++ b/components/permissions/android/permission_dialog_delegate.cc
+@@ -63,10 +63,12 @@ void PermissionDialogDelegate::CreateJavaDelegate(
+           permission_prompt_->GetIconId()),
+       ConvertUTF16ToJavaString(env, permission_prompt_->GetMessageText()),
+       primaryButtonText, secondaryButtonText));
++  BRAVE_PERMISSION_DIALOG_DELEGATE_CREATE_JAVA_DELEGATE
```

Comment:
override PermissionDialogDelegate -> BravePermissionDialogDelegate in permission_prompt_android.cc an option?

Reply:
not sure how it can work, we still need to fill some data in `j_delegate_` which is created and immediately used in `PermissionDialogDelegate` constructor.
is it possible to do something in a constructor without patches? `virtual/override` will not work here.

Reply:
removed this patch by stubbing `Java_PermissionDialogController_createDialog` function.